lemarche/utils/apis/api_entreprise.py

```diff
@@ -37,43 +57,44 @@ def entreprise_get_or_error(siren, reason="Inscription au marché de l'inclusion
             "object": reason,
         }
     )
-
-    url = f"{settings.API_ENTREPRISE_BASE_URL}/entreprises/{siren}?{query_string}"
+    url = f"{settings.API_ENTREPRISE_BASE_URL}/insee/sirene/unites_legales/{siren}?{query_string}"
```

Comment:
je pense que ça serait cool d'avoir une fonction du type, ça permettra de un peu de code j'ai l'impression et ça ajoute un peu de lisibilité :) 
```suggestion
    def get_url_endpoint(endpoint:str, query_params:dict):
        query_string = urlencode(query_params)
        return f"{settings.API_ENTREPRISE_BASE_URL}/{endpoint}?{query_string}
    query_params = {
            "recipient": settings.API_ENTREPRISE_RECIPIENT,
            "context": settings.API_ENTREPRISE_CONTEXT,
            "object": reason,
    }
    url = get_url_endpoint(f"/insee/sirene/unites_legales/{siren}", query_params)
```